appengine-java-logging/src/test/java/com/google/apphosting/logging/JsonFormatterTest.java

```diff
@@ -85,19 +104,155 @@ public void messageIncludesStackTrace() throws Exception {
     BufferedReader reader = new BufferedReader(new StringReader(message));
     assertEquals("logger: message", reader.readLine());
     assertEquals("java.lang.Throwable: thrown", reader.readLine());
-    assertTrue(
-        reader
-            .readLine()
-            .startsWith("\tat " + getClass().getName() + ".messageIncludesStackTrace"));
+    assertTrue(reader.readLine()
+        .startsWith("\tat " + getClass().getName() + ".messageIncludesStackTrace"));
+  }
+
+  @Test
+  public void testCommonsLogging() {
+    final Instant now = Instant.now();
+    final String expectedThreadName = Thread.currentThread().getName();
+
+    // Generate Events from Commons Logging
+    new CommonsLoggingExample().run();
+
+    // Verify that event was captured by slf4j -> java.util.logging
```

Comment:
Was wanting to keep the example as pristine as possible, let the testcase assert the expected results instead.